[2025-10-22T13:57:37.516+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-10-22T13:57:37.607+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: data_quality.soda_test_core scheduled__2025-10-20T16:00:00+00:00 [queued]>
[2025-10-22T13:57:37.636+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: data_quality.soda_test_core scheduled__2025-10-20T16:00:00+00:00 [queued]>
[2025-10-22T13:57:37.638+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-10-22T13:57:37.711+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): soda_test_core> on 2025-10-20 16:00:00+00:00
[2025-10-22T13:57:37.734+0000] {standard_task_runner.py:63} INFO - Started process 1017 to run task
[2025-10-22T13:57:37.751+0000] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'data_quality', 'soda_test_core', 'scheduled__2025-10-20T16:00:00+00:00', '--job-id', '55', '--raw', '--subdir', 'DAGS_FOLDER/main.py', '--cfg-path', '/tmp/tmpz7jwyfe0']
[2025-10-22T13:57:37.753+0000] {standard_task_runner.py:91} INFO - Job 55: Subtask soda_test_core
[2025-10-22T13:57:38.062+0000] {task_command.py:426} INFO - Running <TaskInstance: data_quality.soda_test_core scheduled__2025-10-20T16:00:00+00:00 [running]> on host 5adcf1b12c4a
[2025-10-22T13:57:38.428+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='gidiboateng200@gmail.com' AIRFLOW_CTX_DAG_OWNER='dataengineers' AIRFLOW_CTX_DAG_ID='data_quality' AIRFLOW_CTX_TASK_ID='soda_test_core' AIRFLOW_CTX_EXECUTION_DATE='2025-10-20T16:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-10-20T16:00:00+00:00'
[2025-10-22T13:57:38.433+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-10-22T13:57:38.443+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-10-22T13:57:38.447+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'soda scan -d pg_datasource -c /opt/airflow/include/soda/configuration.yml -v SCHEMA=core /opt/airflow/include/soda/checks.yml']
[2025-10-22T13:57:38.485+0000] {subprocess.py:86} INFO - Output:
[2025-10-22T13:57:42.885+0000] {subprocess.py:93} INFO - [13:57:42] Soda Core 3.3.14
[2025-10-22T13:57:43.152+0000] {subprocess.py:93} INFO - [13:57:43] Scan summary:
[2025-10-22T13:57:43.154+0000] {subprocess.py:93} INFO - [13:57:43] 4/4 checks PASSED:
[2025-10-22T13:57:43.154+0000] {subprocess.py:93} INFO - [13:57:43]     yt_api in pg_datasource
[2025-10-22T13:57:43.155+0000] {subprocess.py:93} INFO - [13:57:43]       Check for Like_Count greater than Video_Views [PASSED]
[2025-10-22T13:57:43.156+0000] {subprocess.py:93} INFO - [13:57:43]       Check for Comments_Count greater than Video_Views [PASSED]
[2025-10-22T13:57:43.157+0000] {subprocess.py:93} INFO - [13:57:43]       missing_count("Video_ID") = 0 [PASSED]
[2025-10-22T13:57:43.158+0000] {subprocess.py:93} INFO - [13:57:43]       duplicate_count("Video_ID") = 0 [PASSED]
[2025-10-22T13:57:43.158+0000] {subprocess.py:93} INFO - [13:57:43] All is good. No failures. No warnings. No errors.
[2025-10-22T13:57:43.884+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-10-22T13:57:43.886+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-10-22T13:57:43.942+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=data_quality, task_id=soda_test_core, run_id=scheduled__2025-10-20T16:00:00+00:00, execution_date=20251020T160000, start_date=20251022T135737, end_date=20251022T135743
[2025-10-22T13:57:44.037+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-10-22T13:57:44.074+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-10-22T13:57:44.078+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
